code/modules/mob/living/simple_mob/subtypes/vore/pakkun.dm

```diff
@@ -46,8 +47,46 @@
 	projectilesound = 'sound/effects/slime_squish.ogg'
 
 	ai_holder_type = /datum/ai_holder/simple_mob/ranged/pakkun
+	vore_default_mode = DM_SELECT
 
 	var/extra_posessive = FALSE					// Enable if you want their tummy hugs to be inescapable
+	var/autorest_cooldown = 100
+
+/mob/living/simple_mob/vore/pakkun/Life()
+	. = ..()
+	if(autorest_cooldown)
+		autorest_cooldown --
+	else if(prob(5) && ai_holder.stance == STANCE_IDLE)
```

Comment:
will this still trigger if there's a client? if it does then get_AI_stance() returns null if there's a client and ai_holder.stance otherwise.